### ISSUE TITLE
less/scss preprocessors not producing css in 1.4.2

### ISSUE THREAD

**Issue by eymiha** [meteor/meteor#8008]
In 1.4.2, I added the less package and created a file `app.less` under client containing

    body { margin: 0 }

ran meteor, and no css was emitted. I removed the package and added fourseven:scss, changed the file to `app.scss`, ran meteor and again, no css was emitted.

I created a new project in 1.3.4, added the less package and copied the old `app.scss` to `app.less` under my new projects client. I ran meteor and voila, css was created from the less file.

**Comment by benjamn** [comment]
How are you determining if CSS was emitted? For me, these reproduction steps result in a `<link>` tag as the first child element of the `<head>`, something like:
```html
<head>
  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="/merged-stylesheets.css?hash=2809b331c0316963f4f876a8e1e7f1fc667ba0ee">
  ...
```
If you visit http://localhost:3000/merged-stylesheets.css?hash=... in your browser, the processed CSS should be visible there.

Perhaps try a CSS rule whose effects are a little more obvious, and/or less vulnerable to being overridden elsewhere?
```css
body {
  background: lightgreen;
}
```

**Comment by klaussner** [comment]
I've noticed a similar problem, but only when importing files. Here's a reproduction: https://github.com/klaussner/meteor-css-issue

Since 1.4.2, adding new `.less` files and importing them while `meteor` is running (sometimes) results in `Uncaught Error: Cannot find module './<filename>.less.css'` in the browser console. The files are built correctly when `meteor` is restarted, though.

**Comment by Akryum** [comment]
[Here is the issue](https://github.com/Akryum/meteor-vue-component/issues/80) in the vue-meteor-component project that relates to this (Meteor 1.4.2). Adding new `.vue` files while meter is running throws a not found error.

**Comment by benjamn** [comment]
Please try this again after running
```
meteor update --release 1.4.2.2-beta.0
```
in an existing app, or
```
meteor create --release 1.4.2.2-beta.0 your-new-app
```
to create a new app using the latest beta version.

**Comment by klaussner** [comment]
`.less` and `.scss` imports are working!

**Comment by averissimo** [comment]
Not working for me in `Meteor 1.4.2.3`

What I've done:

- `meteor add less`
- create client/ui/css/app.less with 'body { background-color: #777; }`
- refreshed browser, restarted meteor and it does not add to the `merged-stylesheets.css` file

It detects a change, but does not update files as this appears in meteor console:

`=> Client modified -- refreshing (x10)`

**Comment by averissimo** [comment]
Was not working for me, I was just putting the less file in the wrong directory parent -_-

It must be under 'client` (was using imports).. duhh

**Comment by abernix** [comment]
Closing as this should be resolved by Meteor 1.4.2.3 which is out now.  Happy to reopen if the problem if the problem still persists though.
